Import/index.mjs

```diff
@@ -0,0 +1,61 @@
+import {Manifest} from "manifesto.js"
+import dbDriver from "../database/driver.mjs"
+
+const database = new dbDriver("mongo")
+
+export class Project {
```

Comment:
Careful here.  We need a /classes/Project/Project.mjs.

That class will be able to construct and import TPEN3 Projects.

A service route would use the Project.mjs for doing things like `services.t-pen.org/project/create?manifestURI=https://iiif.io/api/cookbook/recipe/0001-mvm-image/manifest.json`